openstack_controller/datadog_checks/openstack_controller/openstack_controller.py

```diff
@@ -700,7 +708,8 @@ def check(self, instance):
             self.collect_hypervisors_metrics(custom_tags=custom_tags,
                                              use_shortname=use_shortname,
                                              collect_hypervisor_metrics=collect_hypervisor_metrics,
-                                             collect_hypervisor_load=collect_hypervisor_load)
+                                             collect_hypervisor_load=collect_hypervisor_load,
+                                             openstack_sdk_cloud_name=openstack_sdk_cloud_name)
```

Comment:
Would it be possible to have this within the api implementation and have the api function that is not supported raise an exception that we would then catch in the check?